test/unit/plugins.test.js

```diff
@@ -1,206 +1,214 @@
+/* eslint-env qunit */
 import {IE_VERSION} from '../../src/js/utils/browser';
 import registerPlugin from '../../src/js/plugins.js';
 import Player from '../../src/js/player.js';
 import TestHelpers from './test-helpers.js';
 import window from 'global/window';
+import sinon from 'sinon';
 
-q.module('Plugins');
+QUnit.module('Plugins');
 
-test('Plugin should get initialized and receive options', function(){
-  expect(2);
+QUnit.test('Plugin should get initialized and receive options', function() {
+  QUnit.expect(2);
 
-  registerPlugin('myPlugin1', function(options){
-    ok(true, 'Plugin initialized');
-    ok(options['test'], 'Option passed through');
+  registerPlugin('myPlugin1', function(options) {
+    QUnit.ok(true, 'Plugin initialized');
+    QUnit.ok(options.test, 'Option passed through');
   });
 
-  registerPlugin('myPlugin2', function(options){
-    ok(false, 'Plugin initialized and should not have been');
+  registerPlugin('myPlugin2', function(options) {
+    QUnit.ok(false, 'Plugin initialized and should not have been');
   });
 
-  var player = TestHelpers.makePlayer({
-    'plugins': {
-      'myPlugin1': {
-        'test': true
+  const player = TestHelpers.makePlayer({
+    plugins: {
+      myPlugin1: {
+        test: true
       }
     }
   });
 
   player.dispose();
 });
 
-test('Plugin should have the option of being initilized outside of player init', function(){
-  expect(3);
+QUnit.test('Plugin should have the option of being initilized ' +
```

Comment:
I wonder if it's possible to tell eslint to ignore the long lines rule for tests.